app/controllers/bids_controller.rb

```diff
@@ -0,0 +1,13 @@
+class BidsController < ApplicationController
+  include Secured
+
+  def create
+    user_id = current_user.id
+    auction_id = params[:id]
+    body = JSON.parse(request.body.read, symbolize_names: true)
+    price = body[:price]
+
+    bid = Bid.create!(auction_id: auction_id, user_id: user_id, price: price)
```

Comment:
```suggestion
    bid = Bid.create!(auction_id: auction_id, user: current_user, price: price)
```

and remove `user_id` variable above